tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/local.settings.Designer.cs

```diff
@@ -0,0 +1,242 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
This file looks unrelated and probably should be removed.

Reply:
Fixed.

Reply:
looks like this file is still here.